docs/build_docs.sh

```diff
@@ -142,6 +148,35 @@ if [[ -d $docbasepath/build/sphinx ]]; then
     echo ""
 fi
 
+current_release_notes=`ls docs/source/releases/latest/*`
```

Comment:
The new release notes are always in the "latest" folder, even if we are creating a doc build for an actual version release.